app/sidekiq/decision_review/saved_claim_nod_status_updater_job.rb

```diff
@@ -105,13 +105,16 @@ def check_attachments_status(nod, uploads_metadata)
         status = upload['status']
         result = false unless UPLOAD_SUCCESSFUL_STATUS.include? status
 
-        upload_id = upload['id']
-        # Increment StatsD and log only for new errors
-        unless old_uploads_metadata.dig(upload_id, 'status') == ERROR_STATUS
-          StatsD.increment("#{STATSD_KEY_PREFIX}_upload.status", tags: ["status:#{status}"])
+        if status == ERROR_STATUS
```

Comment:
It seems like the conditional is different here where logging is happening upon error status whereas before in the unless condition, it was happening when the status was **not** `ERROR_STATUS`. Is this the intent, just want to check?

Reply:
That's correct - we want it to log only if the status changed to `ERROR_STATUS` but not if it was already `ERROR_STATUS`.